client/state/sites/reducer.js

```diff
@@ -40,6 +41,14 @@ const VALID_SITE_KEYS = Object.keys( sitesSchema.patternProperties[ '^\\d+$' ].p
  */
 export function items( state = {}, action ) {
 	switch ( action.type ) {
+		case WORDADS_SITE_APPROVE_REQUEST_SUCCESS:
```

Comment:
Yes, we updated the `options.wordads` property of `/me/sites`.

I kept it flat, in `state/wordads`, because everything else is like it - it's even "glued together" in sidebar.
I personally hate this and I think we should make better integration.
But folks are of the opinion that for now it should be separate, so I'm following that pattern here.
After the test, if we decide to keep it, I'm happy to integrate better.